### PR TITLE
Add simple popup box and version checking on client

### PR DESCRIPTION
**Pull request by rvstwi**
## Summary

Added functionality for building simple pop up boxes at any time, no need to mess with Ubisoft's convoluted system.  Will contribute to issue #20 for clients.

## Testing

I have tested my compiled build in the following scenarios:

- [X] A client running **my build** against a server running **latest stable version**
- [ ] A client running **latest stable version** against a server running **my build**
- [ ] A client running **my build** against a server running **my build**

### CODE REVIEW

**Comment by rvstwi** [comment]
Would like it if we could take a look at the "removing line break" in OpenRVS.uc that I commented out because it was removing the "4" from "v1.4" fetched from latest release.  Why did it work earlier -- IE why was there a line break and now there isn't?

File: OpenPopUpContent.uc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+class OpenPopUpContent extends UWindowDialogClientWindow;
+
+var R6WindowWrappedTextArea wTextBox;
+var R6WindowButton butURL;
+var string urlString;
+var float fMargin;
+
+function MakeTextBox()
+{
+	wTextBox = R6WindowWrappedTextArea(CreateWindow(class'R6WindowWrappedTextArea',fMargin,fMargin,WinWidth-(fMargin*2),WinHeight-(fMargin*2),self));
```
Comment (willroberts) [inline OpenPopUpContent.uc:10]: Do we need to double fMargin?

**Review by willroberts — APPROVED** [review]

**Comment by willroberts** [comment]
> Would like it if we could take a look at the "removing line break" in OpenRVS.uc that I commented out because it was removing the "4" from "v1.4" fetched from latest release.  Why did it work earlier -- IE why was there a line break and now there isn't?

Looking at it again, I'm not quite sure. Something may have changed in the registry handler since I last tested. Removing the strip is the right call.

File: OpenPopUpContent.uc (hunk 1 above)
Comment (rvstwi) [reply OpenPopUpContent.uc:10]: The base CreateWindow() function in UWindowWindow.uc takes two arguments there, X and Y values.  Ubisoft, when building pop up boxes, used the same fMargin value (4.0) for both.  I kept it consistent with Ubisoft's implementation but of course we could have different values for the X margin and the Y margin.

**Pull request merged**